task0.sql

```diff
@@ -0,0 +1,132 @@
+CREATE DATABASE BanksDB
+CREATE TABLE social_statuses(
+	social_status_id INT IDENTITY(1,1) PRIMARY KEY NOT NULL,
+	type VARCHAR(25)
+);
+
+CREATE TABLE banks(
+	bank_id INT IDENTITY(1,1)  PRIMARY KEY NOT NULL,
+	bank_name VARCHAR(25)
+);
+
+CREATE TABLE accounts(
+	account_id INT IDENTITY(1,1)  PRIMARY KEY NOT NULL,
+	account_number VARCHAR(25),
+	balance REAL
+);
+
+CREATE TABLE clients(
+	client_id INT IDENTITY(1,1) PRIMARY KEY NOT NULL,
+	bank_id INT  NOT NULL,
```

Comment:
Better to connect accounts with banks
You can have multiple account from one user for different banks
And here you can have only one user per one bank